### ISSUE TITLE
Continue scanning

### ISSUE THREAD

**Issue by Ronnos** [tjwoon/csZBar#79]
Hello, i'm trying to build my first cordova app to scan barcodes and save them into a database. Now I trigger the barcode scanner with a button. Is it possible to show the result and start scanning automatically again, like in continues mode? 
Kindly Regards, Ron

**Comment by dcousens** [comment]
How would that feedback to your program?

**Comment by Chuckytuh** [comment]
FYI, one way of passing asynchronously data from native to javascript through cordova that I used is described on [this question](http://stackoverflow.com/questions/35903306/how-problematic-are-hardcoded-callbackids) that I asked a while ago. So far I haven't had any problems with it.

**Comment by noveweb** [comment]
Ron, that's solved? I have same problem. Help!

**Comment by dcousens** [comment]
ping @Ronnos

**Comment by Ronnos** [comment]
No, I didn't solve my problem. I gave up on this... Because the application didn't work. The barcode scanner seemed to stuck every second when I was running next loop of my script. I now use a barcode scanning app with an api call to my webservices.
